src/app/app.module.ts

```diff
@@ -32,7 +32,7 @@ declare const location: any;
 if (typeof location.origin === 'undefined') {
   location.origin = location.protocol + '//' + location.host;
 }
-LoopBackConfig.setBaseURL(location.origin);
+// LoopBackConfig.setBaseURL(location.origin);
```

Comment:
I was running webpack on production mode with a watch flag so that it'd compile into the dist folder, and that was symlinked to `client-dist` or something to that effect in the loopback backend, so that I could serve the client/frontend files from the same url as the loopback backend to make raven login less complicated with redirecting between two different domains.